.github/CODEOWNERS

```diff
@@ -61,6 +61,7 @@ packages/kbn-eslint-plugin-eui-a11y @elastic/obs-ux-infra_services-team
 packages/kbn-eslint-plugin-i18n @elastic/obs-knowledge-team @elastic/kibana-operations
 packages/kbn-eslint-plugin-imports @elastic/kibana-operations
 packages/kbn-eslint-plugin-telemetry @elastic/obs-knowledge-team
+packages/kbn-eslint-rule-overrides @elastic/kibana-operations
```

Comment:
I originally thought to put threat-hunting as owners but I am transferring to platform team now and I feel like I can definitely use this in the new team as well, so instead kibana-operations looks like the team to own this.

But as I stated in the description I fully volunteer to help maintain this if you accept this into your files.

@elastic/kibana-operations